torch/_inductor/graph.py

```diff
@@ -877,6 +877,19 @@ def debug(msg):
                         stride_order = ir.NHWC_STRIDE_ORDER
                     result = ir.ExternKernel.require_stride_order(result, stride_order)
 
+            def has_fixed_layout_tag(user):
+                t = user.target
+                return (
+                    isinstance(result, TensorBox)
+                    and isinstance(t, torch._ops.OpOverload)
+                    and (torch._C.Tag.needs_fixed_layout in t.tags)
+                )
+
+            if any(has_fixed_layout_tag(u) for u in n.users):
+                result = ir.ExternKernel.require_stride_order(
+                    result, ir.get_stride_order(n.meta["val"].stride())
+                )
```

Comment:
Can we add the restriction at the use instead of the def ?

If you have:
```
def foo(x):
   t = conv(x) # outputs channels last
   

   fixed_layout_contiguous_op(t)
   conv(t) # requires channels last
```

We'd be doing an extra conversion of t from Channels_Last->Contiguous->Channels last, as opposed to just Channels_Last->Contiguous.

Reply:
Hmm, I think I can do this by checking for the tag before the fallback handler. Let me give it a shot